howto/update/upgrade-anbox.md

```diff
@@ -109,7 +111,13 @@ To upgrade all charms, run the following commands:
     juju refresh --channel=1.20/stable anbox-stream-gateway
     juju refresh --channel=1.20/stable anbox-stream-agent
     juju refresh --channel=1.20/stable coturn
-    juju refresh nats
+
+[note type="information" status="Note"]
+Since the NATS charm has been overhauled to use the modern charm framework (Ops Framework), a new charm source is needed to upgrade to its latest version. The charm source can be switched or replaced using the following command:
```

Comment:
Does this mean that after the first time of source update, user does not need to run `juju refresh nats` every time they upgrade Anbox?

Reply:
Yes, I feel that will make it more obvious what to do for the first time and then what to do for later upgrades.